libevm/JitVM.cpp

```diff
@@ -14,11 +13,135 @@ namespace dev
 {
 namespace eth
 {
+namespace
+{
+
+static_assert(sizeof(Address) == sizeof(evm_hash160),
```

Comment:
I pretty sure static_assert is supported by VS 2013. However, I've seen some bugs there. Nevertheless, I hope we will be able to test it in the CIs. In case of trouble, we can disable that on Windows.